antora-playbook.yml

```diff
@@ -96,12 +96,13 @@ asciidoc:
     prod-operator-image-name: che-operator
     prod-prev-ver-major: 6
     prod-prev-ver: 7.18
+    che-ver: 7.20.2
```

Comment:
this version does not exist. See options at https://github.com/eclipse/che-operator/tree/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/

Reply:
Thx, for the link, updating .)